.github/workflows/release-homebrew.yaml

```diff
@@ -0,0 +1,164 @@
+# When a third-party action is added (i.e., `uses`), please also add it to `download-licenses` in Makefile.
+# When the installers are ready in installer private bucket, run this workflow to test them.
+# TODO: Add job of uploading installers to Github release when installer tests are stable.
+# "release" runner tag is to make target hosts deterministic for easy clean up.
+# TODO: Remove the "release" runner tag when installer tests are stable.
+name: Release Homebrew
+on:
+  workflow_dispatch:
+
+permissions:
+  id-token: write # This is required for requesting the JWT
```

Comment:
Do we need this token? Unsure if we are accessing any resource that requires an ODIC JWT token.

Reply:
I don't think so. Will remove it.

Reply:
Even for `contents` scope, the [default](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token) access is read for the GITHUB_TOKEN. Maybe we can just remove the `permissions` block?